steps/synopsys-pt-eco/scripts/designer-interface.tcl

```diff
@@ -0,0 +1,52 @@
+#=========================================================================
+# designer_interface.tcl
+#=========================================================================
+# The designer_interface.tcl file is the first script run by PTPX (see the
+# top of ptpx.tcl). It is the interface that connects the scripts with
+# the following:
+#
+# - ASIC design kit
+# - Build system variables
+#
+# Author : Kartik Prabhu
+# Date   : September 21, 2020
+
+
+#-------------------------------------------------------------------------
+# Parameters
+#-------------------------------------------------------------------------
+
+set pt_design_name              $::env(design_name)
+set pt_setup_margin             $::env(setup_margin)
+set pt_hold_margin              $::env(hold_margin)
+
+#-------------------------------------------------------------------------
+# Libraries
+#-------------------------------------------------------------------------
+
+set adk_dir                       inputs/adk
+
+set pt_additional_search_path   $adk_dir
+set pt_target_libraries         stdcells.db
+
+set pt_extra_link_libraries     [join "
+                                      [glob -nocomplain inputs/*.db]
+                                      [glob -nocomplain inputs/adk/*.db]
+                                  "]
+
+# List of library cell buffers
+set pt_buf_list					"BUF_X1 BUF_X2 BUF_X4 BUF_X8 BUF_X16 BUF_X32"
```

Comment:
These probably belong in the adk... maybe we should add it?

Reply:
Yeah agreed, I'll move it to the adk